tests/test_database.py

```
@@ -133,5 +143,83 @@ def get_columns(table_name):
+
+    inspector.get_columns.side_effect = get_columns
+    db = MagicMock()
+    db.execute.return_value.fetchone.return_value = None
+
+    revision = _determine_schema_revision(inspector, db)
+
+    assert revision == NORMALIZE_COLD_STORAGE_ENUM_VALUES_REVISION
+
+
+@patch("alembic.command.upgrade", side_effect=RuntimeError("boom"))
+def test_run_startup_migrations_raises_on_failure(mock_upgrade, db_session, monkeypatch):
```

Ruff (0.15.10)

[warning] Unused function argument: `db_session`

(ARG001)